ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -519,6 +560,12 @@ public void clear_aggregation()
         public static List<string> gene_name_labels = new List<string> { "Primary, e.g. HOG1", "Ordered Locus, e.g. YLR113W" };
         public string[] likely_cleavages = new string[] { "I", "L", "A" };
 
+        public void construct_families()
```

Comment:
Could you change the name of this method to construct_target_and_decoy_families or something to distinguish it from the method in ProteoformCommunity?